packages/playback-core/src/types.ts

```diff
@@ -1,6 +1,5 @@
 import type { Options } from 'mux-embed';
-import type Hls from 'hls.js';
-import type { HlsConfig } from 'hls.js';
```

Comment:
question(non-blocker): should `./hls` export an `HlsConfig` type to replace this?